bond_tables.sql

```diff
@@ -105,4 +105,65 @@ CREATE TABLE api.all_bonds_aims_to_dashboard (
   "updated_at" timestamp
 );
 
+CREATE TABLE api.fdus_2020_bond (
+  "subproject_number" numeric,
+  "subproject_name" text,
+  "subproject_status" text,
+  "unit_code" text,
+  "unit_name" text,
+  "appropriated" numeric,
+  "updated_at" timestamp
+);
+
+
+CREATE TABLE api.subprojects_with_appropriations (
+  "subproject_number" numeric,
+  "subproject_name" text,
+  "subproject_appropriation" numeric,
+  "updated_at" timestamp
+);
 
+CREATE TABLE api.subprojects_with_budget (
+  "subproject_number" numeric,
+  "subproject_name" text,
+  "subproject_status" text,
+  "project_estimate" numeric,
+  "updated_at" timestamp
+);
+
+CREATE TABLE api.fdu_expenses_quarterly (
+  "fund" text,
+  "department" int,
+  "fiscal_quarter" text,
+  "fiscal_year" int,
+  "subproject_number" numeric,
+  "subproject_name" text,
+  "unit_code" text,
+  "unit_long_name" text,
+  "month-year" text,
+  "expenses" numeric,
+  "updated_at" timestamp
+);
+
+CREATE TABLE api.quarterly_spend_plan (
+  "unit_code" text,
+  "unit_name" text,
+  "fiscal_year" int,
+  "quarter" int,
+  "month" str,
+  "spend_plan" numeric,
+  "updated_at" timestamp
+);
+
+CREATE TABLE api.fdu_metadata_quarterly (
+  "subproject_number" numeric,
+  "subproject_name" text,
+  "subprogram_code" text,
+  "subprogram_long_name" text,
+  "program_code" text,
+  "program_long_name" text,
+  "unit_code" text,
+  "unit_long_name" text,
+  "appropriated" numeric,
+  "updated_at" timestamp
+);
```

Comment:
now its my turn to be that person who asks for a new line to get rid of this icon

Reply:
dangit i was going to comment about it too but now see u beat me to the punch. i want to be that person heh